.agent/workflows/create-pr.md

```
@@ -0,0 +1,45 @@
+---
+description: How to create a Pull Request for the Serapeum API project
+---
+
+# Create a Pull Request
+
+Follow these steps every time you open a PR. Missing step 2 will cause the `PR Title Check` CI workflow to fail.
+
+## Steps
+
+1. Ensure your branch is named with the Linear ticket prefix:
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)